packages/components/pages/f-self-exclusion/src/components/SelfExclusion.vue

```diff
@@ -310,6 +318,13 @@ export default {
             return `${dd}-${mm}-${yyyy}`;
         },
 
+        checkState (state) {
+            if (state === 'exclusionPeriodExpired') {
```

Comment:
It looks like you're checking the same thing as your new computed property offers?
```suggestion
            if (this.exclusionPeriodExpired) {
```